ajax/libs/mui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "mui",
+  "filename": "js/mui.min.js",
+  "description": "A effective front-end framework clostest to the native app",
```

Comment:
How about
The most native-speed front-end framework

BTW, you got typo `clostest`